ui/ui-frontend/projects/design-system/src/app/components/inputs/inputs.component.html

```diff
@@ -334,6 +334,62 @@ <h4>Date</h4>
       </mat-form-field>
     </div>
 
+    <div class="d-flex mt-4">
+      <vitamui-common-multiple-options-datepicker
+        [formControl]="datepickerYear"
+        class="mr-4"
+        [pickerType]="'year'"
```

Comment:
Tu peux l'écrire plus simplement (perso, je préfère)
```suggestion
        pickerType="year"
```